src/swell/__init__.py

```diff
@@ -9,4 +9,4 @@
 repo_directory = os.path.dirname(__file__)
 
 # Set the version for swell
-__version__ = '1.9.6'
+__version__ = '1.9.7'
```

Comment:
This is a pretty major change (albeit, not a user-facing one), so this should probably be at least a minor version bump.

```suggestion
__version__ = '1.10.0'
```